gcp_delete_old_images_and_instances.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import sys
+
+
+# Delete images older than 2 weeks, except if an image is the newest image in
+# an image family
+def delete_families():
```

Comment:
This does *not* delete families. It deletes *older* images in families, but never the last image in a family.